src/entities/Place/types.ts

```diff
@@ -44,6 +40,8 @@ export type AggregatePlaceAttributes = PlaceAttributes & {
   user_count?: number
   user_visits?: number
   realms_detail?: Realm[]
+  category_id?: string
+  category_ids?: string[]
```

Comment:
why both?

Reply:
because a query can return a category_id, another query can get all the place categories. I'll merge into `category_ids: string[]`

Reply:
because a query can return a category_id, another query can get all the place categories.